webapp/package.json

```diff
@@ -19,8 +19,10 @@
     "intl": "1.2.5",
     "jasny-bootstrap": "3.1.3",
     "jquery": "3.1.1",
+    "localforage": "1.5.0",
     "marked": "mattermost/marked#8f5902fff9bad793cd6c66e0c44002c9e79e1317",
     "match-at": "0.1.0",
+    "mattermost-redux": "mattermost/mattermost-redux#webapp",
```

Comment:
Because it needs a whole bunch of stuff from my redux webapp branch :P Once that's merged I'll switch it over to master but in the mean time to get testing done it's pointed at the branch